near-contract-standards/src/storage_management/mod.rs

```diff
@@ -1,15 +1,20 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::json_types::U128;
-use near_sdk::serde::Serialize;
+use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::AccountId;
 
-#[derive(Serialize)]
+/// Implements both `serde` and `borsh` serialization.
+/// `serde` is typically useful when returning a struct in JSON format for a frontend.
```

Comment:
We typically don't want these kind of docs to be public (denoted by the `///` comments). They're more of of side note when you look at the code. Public docs should be explaining what the struct should be doing instead, providing a few examples if necessary. If anything just remove these and add a link to the near docs explaining them at the very top:
```
// serialization differences between Borsh & Serde:
// https://www.near-sdk.io/contract-interface/serialization-interface
```

```suggestion
```

Reply:
Unresolving conversation we we can add the snippet Phuong has here to the top of the file as suggested. Thanks!